independent-publisher-connectors/TSheets Quickbooks Time/apiDefinition.swagger.json

```diff
@@ -0,0 +1,654 @@
+                "operationId": "GetProjects",
+                "parameters": [
+                    {
+                        "name": "ids",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "x-ms-summary": "Ids",
+                        "description": "Comma-separated list of project ids.",
+                        "x-ms-visibility": "advanced"
+                    },
+                    {
+                        "name": "jobcode_ids",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "x-ms-summary": "Jobcode Ids",
+                        "description": "Comma-separated list of jobcode ids associated with a project.",
+                        "x-ms-visibility": "advanced"
+                    },
+                    {
+                        "name": "parent_jobcode_id",
+                        "in": "query",
+                        "required": false,
+                        "type": "integer",
+                        "x-ms-summary": "Parent Jobcode ID",
+                        "description": "Id of a project jobcode's parent to use for filtering",
+                        "x-ms-visibility": "advanced"
+                    },
+                    {
+                        "name": "name",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "x-ms-summary": "Name",
+                        "description": "Starts matching from the beginning of the string",
+                        "x-ms-visibility": "advanced"
+                    },
+                    {
+                        "name": "active",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "default": "yes",
+                        "x-ms-summary": "Active",
+                        "enum": [
+                            "yes",
+                            "no",
+                            "both"
+                        ],
+                        "description": "'yes', 'no', or 'both'. Default is 'yes'."
+                    },
+                    {
+                        "name": "by_jobcode_assignment",
+                        "in": "query",
+                        "required": false,
+                        "type": "boolean",
+                        "x-ms-summary": "By Jobcode assignment",
+                        "description": "If specified, only projects with a jobcode_id the user is assigned to will be returned",
+                        "x-ms-visibility": "advanced"
+                    }
+                ]
+            }
+        },
+        "/users": {
+            "get": {
+                "responses": {
+                    "default": {
```

Comment:
@artesianst default responses is not allowed. Please define a proper response, please see [here](https://learn.microsoft.com/en-gb/connectors/custom-connectors/coding-standards#responses)